consensus/istanbul/backend.go

```diff
@@ -36,14 +36,14 @@ type Backend interface {
 	EventMux() *event.TypeMux
 
 	// Broadcast sends a message to all validators (include self)
-	Broadcast(valSet ValidatorSet, payload []byte) error
+	Broadcast(valSet ValidatorSet, code uint64, payload []byte) error
```

Comment:
Q: what is the `code`?

Reply:
The code identifies the type of message `0x12` for `PREPREPARE`, `0x13` for `PREPARE`, `0x14` for `COMMIT` and `0x15` for `ROUNDCHANGE` qbft message. This would be `0x11` for legacy IBFT